src/main/java/com/denizenscript/clientizen/objects/LocationTag.java

```diff
@@ -0,0 +1,203 @@
+package com.denizenscript.clientizen.objects;
+
+import com.denizenscript.clientizen.util.Utilities;
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Position;
+import net.minecraft.util.math.Vec3i;
+
+import java.util.List;
+
+public class LocationTag implements ObjectTag {
+
+    double x, y, z;
+    float yaw, pitch;
+    String world;
+
+    public LocationTag(double x, double y, double z, float yaw, float pitch) {
+        this.x = x;
+        this.y = y;
+        this.z = z;
+        this.yaw = Utilities.normalizeYaw(yaw);
+        this.pitch = pitch;
+    }
+
+    public LocationTag(double x, double y, double z) {
+        this(x, y, z, 0, 0);
+    }
+
+    public LocationTag(Position position) {
+        this(position.getX(), position.getY(), position.getZ());
+    }
+
+    public LocationTag(Position position, float yaw, float pitch) {
+        this(position.getX(), position.getY(), position.getZ(), yaw, pitch);
+    }
```

Comment:
My gut instinct says that everything should be converted to Vec3d instead of taking in a Position

Reply:
I mean, is there a reason not to? `Vec3d` implements `Position`